lib/discordrb.rb

```diff
@@ -120,6 +120,19 @@ def self.timestamp(time, style = nil)
       "<t:#{time.to_i}:#{TIMESTAMP_STYLES[style] || style}>"
     end
   end
+
+  # A utility method to base64 encode a file like object using its mime type.
+  # @param file [File, #read] A file like object that responds to #read.
+  # @return [String] The file object encoded as base64 image data.
+  def self.encode64(file)
```

Comment:
I wonder if we should consider slightly changing this name to better indicate that it's for file data. Open to pushback